helphours/forms.py

```diff
@@ -13,7 +13,8 @@ class JoinQueueForm(FlaskForm):
         Email()
     ])
     eid = StringField('EID', validators=[
-        DataRequired()
+        DataRequired(),
+        Regexp('^[a-zA-Z0-9]+$', message="Not a valid EID token")
```

Comment:
I feel like the error message is a bit misleading but I can't really think of a better option.
I don't think we should say "token" and also saying "it's not a valid EID" seems to imply we either verify these EIDs or that we only want to accept EIDs. Kinda tricky...